egs2/TEMPLATE/asr2/asr2.sh

```diff
@@ -876,6 +876,7 @@ if [ ${stage} -le 7 ] && [ ${stop_stage} -ge 7 ] && ! [[ " ${skip_stages} " =~ [
 
         _opts="--non_linguistic_symbols ${nlsyms_txt}"
 
+        tgt_bpe_train_text="${data_feats}/${train_set}/text.${tgt_case}.${tgt_lang}"
```

Comment:
You may specify this in the `run.sh`.  E.g. https://github.com/espnet/espnet/blob/master/egs2/librispeech/asr2/run.sh#L50C1-L56C3